### PR TITLE
JBPM-6706 - Task related REST api do not respect container-id in URL

### PR DESCRIPTION
**Pull request by mswiderski**

### CODE REVIEW

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by sutaakar** [comment]
@mswiderski The commented tests are accidental leftover?

**Comment by mswiderski** [comment]
Jenkins retest this

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java [hunk 1]
```diff
@@ -96,14 +96,14 @@ protected void addExtraCustomClasses(Map<String, Class<?>> extraClasses)
                                        kieContainer.getClassLoader()));
     }
 
-    @Test
```
Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java]: Please uncomment the tests here.

**Comment by sutaakar** [comment]
Looks good except of the commented tests.

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java (hunk 1 above)
Comment (mswiderski) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java]: uncommented, damn it I did it again ;) thanks for catching it

**Pull request merged**